frontend/common/src/components/Header/Header.tsx

```diff
@@ -57,20 +73,33 @@ const Header: React.FunctionComponent<HeaderProps> = ({ baseUrl, width }) => {
             </a>
           </div>
           <div
-            data-h2-flex-item="base(1of1) p-tablet(1of2)"
-            data-h2-text-align="base(center) p-tablet(right)"
+            data-h2-display="base(flex)"
+            data-h2-flex-direction="base(column) p-tablet(row)"
+            data-h2-gap="base(x.5) p-tablet(x1)"
+            data-h2-align-items="base(center)"
+            data-h2-justify-content="base(center) p-tablet(flex-end)"
+            data-h2-text-align="base(center) p-tablet(left)"
           >
-            <Link
-              href={languageTogglePath}
-              lang={changeToLang === "en" ? "en" : "fr"}
-              title={intl.formatMessage({
-                defaultMessage: "Change language",
-                id: "GowrkB",
-                description: "Title for the language toggle link.",
-              })}
-            >
-              {changeToLang === "en" ? "English" : "Français"}
-            </Link>
+            <div>{/* <ThemeSwitcher /> */}</div>
+            <div>
+              <Link
+                data-h2-background-color="base:focus-visible(focus)"
+                data-h2-outline="base(none)"
+                data-h2-color="base:hover(tm-blue.dark) base:focus-visible(black)"
+                href={languageTogglePath}
+                lang={changeToLang === "en" ? "en" : "fr"}
+              >
+                {intl.formatMessage(
+                  {
+                    defaultMessage:
+                      "<hidden>Changer la langue en </hidden>{lang}",
+                    id: "BJi10o",
+                    description: "Title for the language toggle link.",
+                  },
+                  { lang: changeToLang === "en" ? "English" : "Français" },
```

Comment:
No need to have logic here

```suggestion
                      "<hidden>Changer la langue en </hidden>Français",
                    id: "BJi10o",
                    description: "Title for the language toggle link.",
                  },
```

Reply:
LOL! What was I thinking? 😆